config/ab_tests.yml

```diff
@@ -50,3 +53,6 @@ test:
   2021_10_mobile_search_panel_test:
     present: 1
     overlay: 0
+  2021_11_desktop_search_results_page_test:
+    present: 1
+    alternative: 0
```

Comment:
Doesn't this mean 'always use the present version and never the alternative'?

Reply:
Yes - I believe we set it like that under `test:` so we don't get the variants complicating things when we are running tests?